src/test/java/com/mabl/integration/jenkins/MablStepDeploymentRunnerTest.java

```diff
@@ -75,7 +79,9 @@ public void setup() {
                 false,
                 true,
                 buildPath,
-                envVars
+                envVars,
```

Comment:
Are there any more test cases to cover? Seems the automatic coverage warnings from CI/CD in this PR think we should add some addition cases.